### PR TITLE
Update cli ref for publish command.

### PR DESCRIPTION
**Pull request by artursouza**
Thank you for helping make the Dapr documentation better!

If you are a new contributor, please see the [this contribution guidance](https://docs.dapr.io/contributing/contributing-docs/) which helps keep the Dapr documentation readable, consistent and useful for Dapr users.

Note that you must see that the suggested changes do not break the website [docs.dapr.io](https://docs.dapr.io). See [this overview](https://github.com/dapr/docs/blob/master/README.md#overview) on how to setup a local version of the website and make sure the website is built correctly.

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Update cli ref for publish command, adding publish-app-id.

## Issue reference

_Please reference the issue this PR will close: https://github.com/dapr/cli/issues/536

### CODE REVIEW

File: daprdocs/content/en/reference/cli/dapr-publish.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ dapr publish [flags]
 
 | Name | Environment Variable | Default | Description
 | --- | --- | --- | --- |
+| `--publish-app-id` | `-i`| | The id for publishing application
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/cli/dapr-publish.md:22]: @artursouza Is this the id of the application you're publishing to or a separate id?

Comment (mukundansundar) [reply daprdocs/content/en/reference/cli/dapr-publish.md:22]: A separate ID, its the ID of the application you are using to publish.

Comment (artursouza) [reply daprdocs/content/en/reference/cli/dapr-publish.md:22]: Any recommendation on the wording or is it clear?

Comment (AaronCrawfis) [reply daprdocs/content/en/reference/cli/dapr-publish.md:22]: Maybe "An ID that represents the app from which you are publishing"?

Comment (artursouza) [reply daprdocs/content/en/reference/cli/dapr-publish.md]: Done.

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**